src/ignitus-StudentSignUp/Components/Signup.js

```diff
@@ -14,18 +14,15 @@ class Signup extends React.Component {
     this.setState({ value: e.target.value });
   }
 
-  handleSubmit(e) {
-    e.preventDefault();
-  }
-
   render() {
+    const { value } = this.state;
     return (
       <div className="container _container-custom p-5">
         <div className="row shadow">
           <div className="col-md-6 p-0 container-image">
-            <img className="img-fluid _img-login d-block" src={loginImg} alt="login" />
+            <img className="img-fluid _img-login d-block" src={loginImg} alt="" />
```

Comment:
please readd the alt tag again

Reply:
I didn't realised its already updated, will update then. thanks